src/Akeneo/Tool/Bundle/ElasticsearchBundle/IndexConfiguration/UpdateIndexMapping.php

```diff
@@ -28,7 +28,7 @@ final class UpdateIndexMapping
     public function updateIndexMapping(Client $client, string $indexNameOrAlias, Loader $indexConfiguration): void
     {
         // We don't care about the index name anymore as we use alias
-        $newIndexName = strtolower($indexNameOrAlias . '_' . str_replace('.', '_', CommunityVersion::VERSION) . '_' . Uuid::uuid4());
+        $newIndexName = strtolower($indexNameOrAlias . '_' . '_' . Uuid::uuid4());
```

Comment:
```suggestion
        $newIndexName = strtolower($indexNameOrAlias . '__' . Uuid::uuid4());
```
Or keep just one underscore, I don't really understand if it's important.